tests/driver.C

```diff
@@ -1,5 +1,8 @@
+// Ignore overloaded-virtual warnings coming from cppunit headers
+#include <libmesh/ignore_warnings.h>
 #include <cppunit/extensions/TestFactoryRegistry.h>
 #include <cppunit/ui/text/TestRunner.h>
+#include <libmesh/restore_warnings.h>
```

Comment:
Nice. Just the other day I was seeing new warnings from cppunit headers which I thought I fixed. It's really unbelievable to me how many warnings are generated by that codebase.